Tasks/XamariniOS/task.json

```diff
@@ -160,6 +169,7 @@
         },
         {
             "name": "provProfileUuid",
+            "aliases": [ "signingProvisioningProfileUuid" ],
```

Comment:
It seems like the Label and the alias should simply use "Id" instead of UUID. I.e. signingProvisioningProfileId and Provisioning Profile ID